configs/opencv-ovms/demos/classification/python/classification_demo.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+"""
```

Comment:
remove this

Reply:
that's the quote to comment the license header

Reply:
ok